flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+    "only-arches": ["x86_64"]
```

Comment:
why?

Reply:
I was thinking wheels are locked to arches? So the python deps would have to be reworked to also have an arch path (which I then can't test)

Reply:
NW.js doesn't support ARM it seems https://github.com/nwjs/nw.js/issues/1151